src/timeline/TimelineModel.cpp

```diff
@@ -379,7 +391,7 @@ TimelineModel::data(const QString &id, int role) const
                 else
                         return {};
         case RoomId:
-                return QVariant(QString::fromStdString(room_id(event)));
+                return QVariant(room_id_);
```

Comment:
@deepbluev7 I don't remember why I made this update.  Can you verify if this makes sense, or should I change it back?

Reply:
`room_id | string | Required. The ID of the room associated with this event. Will not be present on events that arrive through /sync, despite being required everywhere else.`

=> Our events may not have a room_id set

https://matrix.org/docs/spec/client_server/latest#get-matrix-client-r0-sync

Reply:
Ah, gotcha, so this was part of the updates that you made.  Thanks!